river/Seat.zig

```diff
@@ -83,10 +89,15 @@ request_set_primary_selection: wl.Listener(*wlr.Seat.event.RequestSetPrimarySele
     wl.Listener(*wlr.Seat.event.RequestSetPrimarySelection).init(handleRequestSetPrimarySelection),
 
 pub fn init(self: *Self, name: [*:0]const u8) !void {
+    const event_loop = server.wl_server.getEventLoop();
+    const mapping_repeat_timer = try event_loop.addTimer(*Self, handleMappingRepeatTimeout, self);
```

Comment:
I debated. The documentation implied timers were only inserted into the event loop while actively waiting to fire. Since there's only one per seat, it seemed the tradeoff of keeping one allocated even if not needed was better than allocating and freeing one at each use.

Reply:
Agreed, it's definitely better to just keep the timer around than constantly alloc and destroy it.